### ISSUE TITLE
Allow clients to specify label styling

### ISSUE THREAD

**Issue by nickgaray** [opensensorhub/osh-js#527]
![image](https://user-images.githubusercontent.com/37518594/135371741-208dd10c-fc6b-4090-9bc0-455c05b3fd83.png)
Label styling, such as color and optionally border should be allowed for any PointMarkerLayer.

**Comment by nickgaray** [comment]
The example image provided shows a black label with white outline

**Comment by mdhsl** [comment]
I agree, the biggest difficulty is to have something homogeneous between the different views. But for now, we can add the properties in the Layer and implement them in the Cesium view

**Comment by alexrobin** [comment]
Instead of doing something specific for Cesium label outlines, I think it would be better to have a general mechanism to add any custom configuration that is specific to a particular view implementation (i.e. not handled by generic styler properties). It could be done via a callback providing the graphical object created by the view impl for example, so it can be further customized.
